// juniper.afi_cap_entry_match is generated by proto_generator as a protobuf
// representation of a YANG schema.
//
// Input schema modules:
//  - afi-cap-action.yang
//  - afi-cap-entry-action.yang
//  - afi-cap-entry-match.yang
//  - afi-cap-entry.yang
//  - afi-cap-match.yang
//  - afi-cap.yang
//  - afi-common.yang
//  - afi-cos-classifier.yang
//  - afi-counter.yang
//  - afi-decap.yang
//  - afi-discard.yang
//  - afi-encap.yang
//  - afi-filter.yang
//  - afi-fwd-sandbox.yang
//  - afi-indexed-list.yang
//  - afi-indirect.yang
//  - afi-list.yang
//  - afi-match.yang
//  - afi-oam.yang
//  - afi-policer.yang
//  - afi-port.yang
//  - afi-receive.yang
//  - afi-replicate.yang
//  - afi-sample.yang
//  - afi-sandbox.yang
//  - afi-selector.yang
//  - afi-switch.yang
//  - afi-table.yang
//  - afi-tree-entry.yang
//  - afi-tree.yang
//  - afi-types.yang
//  - afi.yang
syntax = "proto3";

package juniper.afi_cap_entry_match;

import "ywrapper/ywrapper.proto";
import "yext/yext.proto";
import "juniper/enums/enums.proto";

message AfiCapEntryMatch {
  ywrapper.UintValue destination_ipv4_address = 378476815;
  ywrapper.BytesValue destination_mac_address = 54776669;
  ywrapper.UintValue ethertype = 153235660;
  juniper.enums.AfiCapEntryMatchAfiInetFamily inner_ip_proto = 466296291;
  ywrapper.BoolValue inner_vlan_cfi = 426519941;
  ywrapper.UintValue inner_vlan_dot1p = 105572003;
  ywrapper.UintValue inner_vlan_id = 511870608;
  juniper.enums.AfiCapEntryMatchAfiField ip_protocol = 327635748;
  ywrapper.BoolValue ipv4_ttl = 480696818;
  ywrapper.UintValue l3_interface_class = 75163530;
  ywrapper.UintValue l4_destination_port = 268263093;
  ywrapper.BoolValue l4_packet = 64806877;
  ywrapper.UintValue l4_source_port = 382902254;
  juniper.enums.AfiCapEntryMatchAfiInetFamily outer_ip_proto = 423644108;
  juniper.enums.AfiCapEntryMatchAfiField outer_tpid = 488591893;
  ywrapper.BoolValue outer_vlan_cfi = 489148286;
  ywrapper.UintValue outer_vlan_dot1p = 406427760;
  ywrapper.UintValue outer_vlan_id = 153122173;
  ywrapper.UintValue pipeline_id = 420155160;
  ywrapper.UintValue source_ipv4_address = 358418578;
  ywrapper.BytesValue source_mac_address = 181758066;
  ywrapper.UintValue src_port = 306124650;
  ywrapper.UintValue src_trunk = 325256887;
  ywrapper.IntValue system_port_class = 531911158;
  ywrapper.UintValue tos = 336707838;
  ywrapper.BoolValue tunnel_terminated = 326667972;
  ywrapper.UintValue virtual_slice_group = 193679854;
  ywrapper.BoolValue vlan_inner_present = 274998374;
  ywrapper.BoolValue vlan_outer_present = 373194379;
}
